source/includes/fundamentals/code-snippets/builders/Updates.java

```diff
@@ -0,0 +1,264 @@
+package docs.builders;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.model.UpdateOptions;
+
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.text.ParsePosition;
+import java.text.SimpleDateFormat;
+import java.util.List;
+
+import static com.mongodb.client.model.Updates.*;
+import static com.mongodb.client.model.Filters.*;
+import static com.mongodb.client.model.UpdateOptions.*;
+
+public class Updates {
+
+    private final MongoCollection<Document> collection;
+    private final MongoClient mongoClient;
+    private final MongoDatabase database;
+
+    private Updates() {
+        // begin declaration
+        final String uri = System.getenv("DRIVER_REF_URI");
+
+        mongoClient = MongoClients.create(uri);
+        database = mongoClient.getDatabase("builders");
+        collection = database.getCollection("updates");
+        // end declaration
+    }
+
+    public static void main(String[] args) {
+        Updates updates = new Updates();
+        updates.resetCollection(updates);
+
+        System.out.println("setUpdate:");
```

Comment:
Did you intend to leave the println statements in?

Reply:
Yes, I forgot to include them before and it took me a while to figure out the order of the updates being performed.